app/detail/template.hbs

```diff
@@ -15,7 +15,7 @@
     {{#link-to "index"}}
   <button class="left"><i class="fa fa-chevron-left"></i></button>
   {{/link-to}}
-  {{#link-to "edit" model.id}}
+  {{#link-to "edit" run._id/edit}}
```

Comment:
Shouldn't need `/edit` to the end.

When you make a `link-to` the first argument is the NAME of the route, the arguments after that fill in any dynamic segments.
From there, the Ember router works to build the final URL.

So in here, it just matters that the `edit` route needs a single dynamic segment. This template doesn't care if the URL for that route is `${model.id}/edit` or `${model.id}/some-crazy-nonsense`